src/index.js

```diff
@@ -0,0 +1,40 @@
+import './style.css';
+
+const tasksContainer = document.querySelector('.tasks-container');
+
+const tasks = [
+  {
+    index: 1,
+    description: 'Hello',
+    completed: true,
+  },
+  {
+    index: 2,
+    description: 'World',
+    completed: true,
+  },
+  {
+    index: 3,
+    description: '!!!!',
+    completed: true,
+  },
+];
```

Comment:
- [ ] **Required:** You have rendered the list excellently to the DOM, however, you are required to render them in the order of their index.

**_Hint:_** Consider `Array.sort((a, b) => a.index - b.index`